### ISSUE TITLE
[Bug]: Unknown function : v:bufferline_private.handle_xxx

### ISSUE THREAD

**Issue by AndyWyy** [akinsho/bufferline.nvim#633]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What happened?

I refer to the document for configurate the bufferline, but some problems happend.
Maybe there is wrong with the  **mouse actions**.
I click the mouse after configuration, but errors are always there：
![image](https://user-images.githubusercontent.com/56649489/206632936-372d1d00-0c64-4b5b-998b-b990b62f7e5c.png)
![image](https://user-images.githubusercontent.com/56649489/206632990-a3eb24fe-45cc-4589-b6fc-62ef16ce0aa8.png)

### What did you expect to happen?

Expected behaviour:
It is expected to close the bufferline when click the 'x' .

### Config
```
local status, bufferline = pcall(require, "bufferline")
if not status then
    vim.notify("没有找到 bufferline")
  return
end

-- bufferline 配置
-- https://github.com/akinsho/bufferline.nvim#configuration
bufferline.setup({
  options = {
    -- 关闭 Tab 的命令，这里使用 moll/vim-bbye 的 :Bdelete 命令
    close_command = "Bdelete! %d",
    right_mouse_command = "Bdelete! %d",
    left_mouse_command = function(bufnum)
      require('bufdelete').bufdelete(bufnum,true)
    end,
    -- 侧边栏配置
    -- 左侧让出 nvim-tree 的位置，显示文字 File Explorer
    offsets = {
      {
        filetype = "NvimTree",
        text = "File Explorer",
        highlight = "Directory",
        text_align = "left",

      },
    },
    -- 使用 nvim 内置 LSP  后
    --diagnostics = "nvim_lsp",
    -- 可选，显示 LSP 报错图标
    ---@diagnostic disable-next-line: unused-local
    diagnostics_indicator = function(count, level, diagnostics_dict, context)
      local s = " "
      for e, n in pairs(diagnostics_dict) do
        local sym = e == "error" and " " or (e == "warning" and " " or "")
        s = s .. n .. sym
      end
      return s
    end,
  },
})
```

### Additional Information

...

**Comment by akinsho** [comment]
@AndyWyy this issue is lacking a lot of information so it isn't really clear exactly what might be happening. Please make sure you are using the latest version of neovim stable and the latest version of this plugins

**Comment by AndyWyy** [comment]
> close_command = "Bdelete! %d",
>     right_mouse_command = "Bdelete! %d",
>     left_mouse_command = function(bufnum)
>       require('bufdelete').bufdelete(bufnum,true)
>     end,

thanks for your reply :)
In short，these configurations about the **mouse action** do not work .
```
    close_command = "Bdelete! %d",
    right_mouse_command = "Bdelete! %d",
    left_mouse_command = function(bufnum)
      require('bufdelete').bufdelete(bufnum,true)
    end,
```
And these errors will be thrown: v:lua.___bufferline_private.handle_xxx 
By the way , I am sure that I am using the latest version of neovim stable and the latest version of this plugins.

**Comment by vennahlot** [comment]
@AndyWyy Had exactly the same problem after a plugin update ~1 month ago with packer. Was using neovim v0.7.2. The problem is resolved after updating to neovim v0.8.1.

**Comment by AndyWyy** [comment]
> @AndyWyy Had exactly the same problem after a plugin update ~1 month ago with packer. Was using neovim v0.7.2. The problem is resolved after updating to neovim v0.8.1.

Thanks a lot！Solved my problem perfectly ～
